frontend/model/state.js

```diff
@@ -145,9 +145,10 @@ const getters = {
   },
   // list of group names and contractIDs
   groupsByName (state) {
-    return Object.entries(store.state.contracts)
-      .filter(([key, value]) => value.type === 'GroupContract')
-      .map(([key]) => ({ groupName: state[key].groupName, contractID: key }))
+    const { contracts } = store.state
+    return Object.keys(contracts)
+      .filter((key) => contracts[key].type === 'GroupContract')
+      .map((key) => ({ groupName: state[key].groupName, contractID: key }))
```

Comment:
Changed it to the Object.keys() version only because it fixes the flow error even though using Object.entries() is obviously simpler and cleaner.
please refer to :

https://github.com/facebook/flow/issues/5838
https://github.com/facebook/flow/blob/master/lib/core.js#L48

Reply:
Very informative comment/explanation, thank you for explaining why this was changed and linking to the appropriate Flow issue.

Can you insert a 1-line comment linking to that flow issue on line 148 explaining that? (So that any developers reading this who are tempted to rewrite it back to `.entries` understand why `.keys` is being used?

Reply:
Thank you for reviewing and I will do that :)